### DISCUSSION TITLE
what is bem

### DISCUSSION THREAD

**Discussion by Pipertzion** [getbem/getbem.github.io#306] [General]
hi there, do i download bem ? 
best regards,

**Comment by vacool6** [comment]
BEM is just a naming pattern for organizing and naming CSS classes. It helps to write clean CSS by following some simple rules https://getbem.com/

Similar to naming variables you can follow
let a =  camelCasing 
let b = snake_case 
let c =  PascelCasing  

BEM is one of many patterns for naming CSS classes.
